### PR TITLE
Improve Pattern: Unified Source Code Inventory

### PR DESCRIPTION
**Pull request by spier**
I reviewed the Unified Source Code Inventory pattern, and made various formatting fixes.

Changes in this PR:

- **(major)** pitching a new Patlet (limited to 2 sentences. slightly shorter)
- split properties Hosting vendor and Hosting type into separate bullets (improves readability; also matches the screenshot more closely)
- removing some unnecessary punctuations
- reformatting Known Instances, to make it easier to add further orgs
- capitalize spelling of 'Organization' everywhere (except in the Patlet)
- Split up the two components that make up this the Solution in the Rationale section

Besides these changes, I am wondering if the [questions](https://github.com/InnerSourceCommons/InnerSourcePatterns/pull/362/files#diff-2c2556858b279ad8a7e7a25185810269c0049f55785d8a506e1c6a06ecca02b9R16-R20) currently listed under **Problem** could be moved into the **Context** section? Context is meant to answer "Do I have this same particular situation?", and it seems that this is the goal of these questions as well.

### CODE REVIEW

**Comment by spier** [comment]
@dterol23 would you mind reviewing this PR?

**Review by dterol23 — APPROVED** [review]
Thanks. I really like the split up of the patlet to make it more clear. I think I just found a small type when reviewing:

Large organization with many legal entities often don't have visibility

I think it should be either:

Large organization**s** with many legal entities often don't have visibility

or

Large organization with many legal entities often **doesn't** have visibility

File: patterns/1-initial/source-code-inventory.md [hunk 1]
```diff
@@ -4,13 +4,14 @@ Unified Source Code Inventory
 
 ## Patlet
 
-In a large organization with different legal entities is often hard to get full visibility into all software assets, in particular all source code. This situation reduces the opportunities to increase business value and keep liability costs, such as software maintenance, under control across the organization as a whole. An organization-level source code inventory addresses these issues while exploiting opportunities to identify and support valuable InnerSource assets.
+Large organization with many legal entities often don't have visibility into all of their software assets (repositories), limiting their ability to increase businesses value and control liability costs holistically.
```
Comment (spier) [inline patterns/1-initial/source-code-inventory.md]:
```suggestion
Large organizations with many legal entities often don't have visibility into all of their software assets (repositories), limiting their ability to increase businesses value and control liability costs holistically.
```

**Comment by spier** [comment]
> I think it should be either:
> 
> Large organization**s** with many legal entities often don't have visibility

Good point. I fixed this.

**Pull request merged**